xml/obs_authorization_token.xml

```diff
@@ -84,6 +84,16 @@
    <screen><command>osc</command> token --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
    <screen><command>osc</command> token --trigger &lt;TOKEN&gt;</screen>
    <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/runservice</screen>
+
+   <para>A token to rebuild a package can be created and trigger by </para>
+   <screen><command>osc</command> token --opertion rebuild --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
+   <screen><command>osc</command> token --operation rebuild --trigger &lt;TOKEN&gt;</screen>
+   <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/rebuild</screen>
+
+   <para>A token to release a package can be created and trigger by </para>
+   <screen><command>osc</command> token --opertion release --create &lt;PROJECT&gt; &lt;PACKAGE&gt;</screen>
+   <screen><command>osc</command> token --operation release --trigger &lt;TOKEN&gt;</screen>
+   <screen><command>curl</command> -H "Authorization: Token &lt;TOKEN&gt;" -X POST /trigger/release</screen>
```

Comment:
Same here, combine all the `screen`s and introduce `replaceable`.

Is `--opertion` a typo? Shouldn't it be named `--operation`?

Reply:
yes...